cmd/package.go

```diff
@@ -523,8 +523,20 @@ func DescribePackage(repo *util.Repo, packageName string) error {
 
 	var pkg *core.Package
 	var cmdConf *core.CmdConfig
+	var readme string
+
+	var tarReader *tar.Reader
+
+	// Load package (tar.gz or tar supported).
```

Comment:
Would it be possible to share parts of the code where we already are using this mechanism to read a package? Perhaps by means of a separate utility function that would create just a `tarReader` that is then used in whatever way needed?